src/brain/githubMetrics/index.test.ts

```diff
@@ -76,10 +81,10 @@ describe('github webhook', function () {
   });
 
   it('does not call insert on dry run', async function () {
```

Comment:
nit: maybe worth doing the test again for 1/0 rather than true/false, since we now support both variants?

Reply:
I actually couldn't get that to work since I believe there's something finnicky around env variables being read into the config once during initialization. I spent a while trying to get it to work then gave up so decided not to go down that route since it's not superrr important